### PR TITLE
[WIP] more require work

### PR DESCRIPTION
**Pull request by janpio**
This is #11507 born again

### Checklist
- [ ] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [ ] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [ ] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I've updated the documentation if necessary.

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Pull Request is Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: deliver/lib/deliver/runner.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-require 'precheck/runner'
-require 'fastlane_core/configuration/configuration'
-require 'fastlane_core/ipa_upload_package_builder'
-require 'fastlane_core/pkg_upload_package_builder'
-require 'fastlane_core/itunes_transporter'
-require 'spaceship'
+require_relative internal('precheck/options')
+require_relative internal('precheck/runner')
+require_relative internal('fastlane_core/configuration/configuration')
+require_relative internal('fastlane_core/crash_reporter/crash_reporter')
```
Comment (janpio) [inline deliver/lib/deliver/runner.rb]: investigate where this comes from

File: fastlane/lib/fastlane/cli_tools_distributor.rb [hunk 2]
```diff
@@ -19,32 +19,6 @@ def running_init_command?
       def take_off
         before_import_time = Time.now
 
-        if !ENV["FASTLANE_DISABLE_ANIMATION"]
```
Comment (janpio) [inline fastlane/lib/fastlane/cli_tools_distributor.rb]: do we really want to remove this?

File: fastlane/lib/fastlane/module.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require_relative '../fastlane_core/lib/fastlane_core/require_relative_helper'
```
Comment (janpio) [inline fastlane/lib/fastlane/module.rb]: check if this is the current state of the module stuff that got removed from fastlaner.rb

File: fastlane/lib/fastlane/setup/setup.rb [hunk 4]
```diff
@@ -351,5 +351,12 @@ def show_analytics_note
   end
 end
 
-require 'fastlane/setup/setup_ios'
-require 'fastlane/setup/setup_android'
+require_relative internal('fastlane/setup/setup_ios')
```
Comment (janpio) [inline fastlane/lib/fastlane/setup/setup.rb]: why those additional require_relatives?

File: fastlane/lib/fastlane/swift_lane_manager.rb [hunk 5]
```diff
@@ -221,9 +221,9 @@ def self.link_user_configs_to_project(updated_message: nil)
       return project_modified
     end
 
-    def self.start_socket_thread(port: nil)
-      require 'fastlane/server/socket_server'
-      require 'fastlane/server/socket_server_action_command_executor'
+    def self.start_socket_thread
```
Comment (janpio) [inline fastlane/lib/fastlane/swift_lane_manager.rb]: missing param

File: fastlane_core/lib/fastlane_core/ui/implementations/shell.rb [hunk 6]
```diff
@@ -26,6 +26,8 @@ def log
         "#{format_string(datetime, severity)}#{msg}\n"
       end
 
+      require_relative internal('fastlane_core/ui/disable_colors') if FastlaneCore::Helper.colors_disabled?
```
Comment (janpio) [inline fastlane_core/lib/fastlane_core/ui/implementations/shell.rb]: additional require_relative?

File: spaceship/lib/spaceship/portal/portal.rb [hunk 7]
```diff
@@ -1,17 +1,16 @@
-require 'spaceship/portal/portal_base'
-require 'spaceship/portal/app'
-require 'spaceship/portal/website_push'
-require 'spaceship/portal/app_group'
-require 'spaceship/portal/app_service'
-require 'spaceship/portal/certificate'
-require 'spaceship/portal/cloud_container'
```
Comment (janpio) [inline spaceship/lib/spaceship/portal/portal.rb]: missing below

File: spaceship/lib/spaceship/tunes/iap.rb [hunk 8]
```diff
@@ -1,13 +1,12 @@
-require 'spaceship/tunes/errors'
```
Comment (janpio) [inline spaceship/lib/spaceship/tunes/iap.rb]: missing below

File: supply/lib/supply.rb [hunk 9]
```diff
@@ -1,29 +1,9 @@
 require 'json'
-require 'supply/options'
-require 'supply/client'
-require 'supply/listing'
-require 'supply/apk_listing'
-require 'supply/uploader'
 
-require 'fastlane_core'
```
Comment (janpio) [inline supply/lib/supply.rb]: missing below

File: bin/fastlane [hunk 10]
```diff
@@ -4,7 +4,8 @@ if RUBY_VERSION < '2.0.0'
   abort("fastlane requires Ruby 2.0.0 or higher")
 end
 
-require_relative '../fastlane/lib/fastlane/cli_tools_distributor'
```
Comment (janpio) [inline bin/fastlane]: undo later

**Comment by milch** [comment]
Ha, I didn't see this before opening #13474

**Comment by janpio** [comment]
Haha, no harm done. If you want you can go through my comments here and see if they apply to your new PR.

**Pull request closed without merging**